examples/flagsmith-on-ecs/terraform/04_loadbalancer.tf

```diff
@@ -0,0 +1,43 @@
+# Production Load Balancer
+resource "aws_lb" "production" {
+  name               = "${local.ecs_cluster_name}-alb"
+  load_balancer_type = "application"
+  internal           = false
+  security_groups    = [aws_security_group.load-balancer.id]
+  subnets            = module.vpc.public_subnets
+}
+
+# Target group
+resource "aws_alb_target_group" "default-target-group" {
+  name        = "${var.app_environment}-${var.app_name}-tg"
+  port        = 8000
+  protocol    = "HTTP"
+  vpc_id      = module.vpc.vpc_id
+  target_type = "ip"
+
+  health_check {
+    path                = var.health_check_path
+    port                = "traffic-port"
+    healthy_threshold   = 5
+    unhealthy_threshold = 2
+    timeout             = 2
+    interval            = 5
+    matcher             = "200"
+  }
+}
+
+
+# Listener (redirects traffic from the load balancer to the target group)
+resource "aws_alb_listener" "ecs-alb-http-listener" {
+  load_balancer_arn = aws_lb.production.id
+  port              = "443"
```

Comment:
I think that the load balancer SG allows 443 and 80, but there's only a 443 listener.

Suggest either:
- 443 only on the listener, 443 only on the load balancer SG
- 443 on the listener that forwards to the workload, 80 on the listener that redirects to 443, 443 and 80 on the load balancer SG (probably with a comment to say that 80 is only used for redirecting to 443)